tests/AgDatabaseMove.Unit/BackupOrder.cs

```diff
@@ -215,6 +276,17 @@ public void DuplicateFiles()
       Assert.Equal(backups.GroupBy(b => b.PhysicalDeviceName).Count(), chain.Count);
     }
 
+    [Fact]
+    public void SingleServerChainEquality()
+    {
+      var list = GetBackupListSingleServer();
+      var agDb = new Mock<IAgDatabase>();
+      agDb.Setup(db => db.RecentBackups()).Returns(list);
+      var chain = new BackupChain(agDb.Object).OrderedBackups.ToList();
+      list.Reverse();
+      Assert.Equal(list, chain);

```

Comment:
Have a look at how we are asserting in the other test cases. Would be better to keep this consistent with those.

If this check-function needs changing, that's fine:
`VerifyListIsAValidBackupChain(backupChain.OrderedBackups.ToList());`